.github/workflows/e2e-testbench.yaml

```diff
@@ -0,0 +1,69 @@
+name: Run E2E Tests
+
+on:
+  workflow_dispatch:
+    inputs:
+      maxTestDuration:
+        description: 'Test duration (Eg: PT2H, P3D)'
+        required: true
+        default: 'PT1D'
+        type: string
+      generation:
+        description: 'Specifies the generation template which should be used by the testbench run'
+        required: false
+        default: 'Zeebe SNAPSHOT'
+        type: string
+      branch:
```

Comment:
:x: I think you don't need the branch input, workflow_dispatch always has the branch or tag, which you access using `GITHUB_REF`. See https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch and https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#workflow_dispatch.

Reply:
:+1: used `github.ref_name`